src/geo/data-providers/geojson/histogram-geojson-data-provider.js

```diff
@@ -27,7 +27,7 @@ HistogramGeoJSONDataProvider.prototype.getData = function () {
     start = typeof options.start === 'number' ? options.start : filter.getMin(columnName);
     width = (end - start) / options.bins;
     values = filter.getValues(false, columnName);
-    bins = d3.layout.histogram().bins(numberOfBins)(values.map(function (f) {
+    bins = d3.layout.histogram().range([start, end]).bins(numberOfBins)(values.map(function (f) {
```

Comment:
We weren't applied the necessary range in order to calculate properly the buckets.
Boy scout fixing!